website_odoo_debranding/views/disable_odoo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <!-- Remove "Create a free website with Odoo" -->
+    <template id="layout_footer_copyright"
+              inherit_id="website.layout_footer_copyright"
+              name="Remove footer Odoo link" priority="20">
+        <xpath expr="//div[@class='pull-right']" position="replace">
+            <div> </div>
```

Comment:
IMO you should also add the same `pull-right` class to it that the original div in order to maintain some compatibility with modules that may be looking for this div.